### ISSUE TITLE
has the paper been accepted?

### ISSUE THREAD

**Issue by adrianrivadulla** [0todd0000/nlreg1d#1]
Hi,

The README suggests waiting until the paper is accepted because the methods have been challenged during review. I am guessing that this is an old commit and hasn't been updated since? 

https://www.sciencedirect.com/science/article/pii/S0021929022001026

Is this the paper it's referring to?

Cheers

**Comment by 0todd0000** [comment]
Yes, that is indeed the paper. Sorry for not updating the README file!  I just updated the README file and removed the "under-review" warning.
